code/modules/projectiles/guns/projectile/automatic.dm

```diff
@@ -273,7 +273,7 @@
 
 // Bulldog shotgun //
 /obj/item/gun/projectile/automatic/shotgun/bulldog
-	name = "\improper 'Bulldog' Shotgun"
+	name = "\improper AS-14 'Bulldog' Shotgun"
 	desc = "A compact, mag-fed semi-automatic shotgun for combat in narrow corridors, nicknamed 'Bulldog' by boarding parties. Compatible only with specialized 8-round drum magazines."
```

Comment:
```suggestion
	desc = "A compact semi-automatic shotgun for combat in narrow corridors, nicknamed 'Bulldog' by boarding parties. Compatible only with specialized 8-round drum magazines."
```
Redundant mag-fed removed. The second sentence makes it obvious that it is fed from magazines.